server/directory/views.py

```diff
@@ -21,4 +22,17 @@ def get(self, request):
             )
 
         serializer = DirectoryMemberSerializer(members, many=True)
-        return Response(serializer.data)
\ No newline at end of file
+        return Response(serializer.data)
+
+class MemberDirectoryView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        # get id from path parameter
+        id = request.query_params.get('id')
+
+        # get Member object with member.user.id == id
```

Comment:
nit: ditto with this comment. "Member.object.get(user_id)" is exactly what this comment says 😛

Reply:
I despise the field lookup syntax and probably won't remember how it works a few months out of working with it.

Reply:
however, if we end up writing a ton more views doing this i'll probably remove the comment eventually